packages/storybook/index.d.ts

```diff
@@ -0,0 +1,21 @@
+export { html } from 'lit-html';
+
+// NB: @types/storybook__polymer doesn't yet exist
+// export { storiesOf, addParameters } from '@storybook/polymer';
```

Comment:
anything we can do about this? the "types" would be the same as for `@storybook/foo`